cmd/podman/common/specgen.go

```diff
@@ -655,26 +655,20 @@ func FillOutSpecGen(s *specgen.SpecGenerator, c *ContainerCLIOpts, args []string
 	return nil
 }
 
-func makeHealthCheckFromCli(inCmd, interval string, retries uint, timeout, startPeriod string) (*manifest.Schema2HealthConfig, error) {
+func makeHealthCheckFromCli(inCmd string, interval string, retries uint, timeout, startPeriod string) (*manifest.Schema2HealthConfig, error) {
+	cmdArr := strings.Split(inCmd, " ")
```

Comment:
```suggestion
	cmdArr := strings.Fields(inCmd)
```
Will handle more inputs correctly.